docker-compose-wsl.yml

```diff
@@ -0,0 +1,75 @@
+version: "3"
```

Comment:
what does `wsl` stand for - wondering if this could be called `-windows` to make it clearer?

Reply:
"_WSL_" stands for [_Windows Subsystem for Linux_](https://docs.microsoft.com/en-gb/windows/wsl/). 

My assumption with using the acronym, is that people using Docker Desktop for Windows, will know what it abbreviates.

Reply:
The problem there is that 99% of the people using the project won't have a clue when to use the docker compose file and what wsl means.

Reply:
> "..._99% of the people using the project won't have a clue_..."

Even though _[the instructions explicitly instruct](https://github.com/boydingham/prisoner-offender-search/tree/boyd-DT-2209-search-running-in-docker#running-prisoner-offender-search-in-docker)_: "_When using Docker Desktop for Windows, do "docker compose -f docker-compose-wsl.yml up_" instead"?

I think you underestimate us "_Windoze_" users ;)